### ISSUE TITLE
latest version can't change different ball at low percent.

### ISSUE THREAD

**Issue by loveinpin** [j-e-k/poketrainer#384]
I figured that latest bot won't choose type of ball at lowest PM

[    pgoapi] [ INFO] Catching Pokemon: Type: Clefairy CP: 504, IV: 37.78, Lvl: 17.0, LvlWild: 17.0, MaxCP: 30, Score: 504, IV-Norm.: 35
[    pgoapi] [ INFO] Attempting catch with POKE_BALL at 21.79% chance. Try Number: 1
[    pgoapi] [ INFO] Caught Pokemon:  Type: Clefairy CP: 504, IV: 37.78, Lvl: 17.0, LvlWild: 17.0, MaxCP: 30, Score: 504, IV-Norm.: 35

config 
```
        "USE_POKEBALL_IF_PERCENT": 75,
        "USE_GREATBALL_IF_PERCENT": 45,
        "USE_ULTRABALL_IF_PERCENT": 15,
```

pgoapi seems can't get info from customs config 

pgoapi
```
        pokeball_percent = config.get("CAPTURE", {}).get("USE_POKEBALL_IF_PERCENT", 15)
        greatball_percent = config.get("CAPTURE", {}).get("USE_GREATBALL_IF_PERCENT", 15)
        ultraball_percent = config.get("CAPTURE", {}).get("USE_ULTRABALL_IF_PERCENT", 15)
```
but the older demo -- ball type could change automatically 

[    pgoapi] [ INFO] Catching Pokemon: Nickname: , Type: Snorlax, CP: 759, IV: 57.7777777778
[    pgoapi] [ INFO] Attempting catch with ball type 3  at 36.03 % chance. Try Number: 1
[    pgoapi] [ INFO] Attempting catch with ball type 3  at 36.03 % chance. Try Number: 2
[    pgoapi] [ INFO] Caught Pokemon:  Nickname: , Type: Snorlax, CP: 759, IV: 57.7777777778

**Comment by rirze** [comment]
What branch and commit are you working with?

**Comment by loveinpin** [comment]
latest  a46a3f176bd838d16212a24d7bffcd1d4418b482
older  4cb22727e521fecb7dd4570294bb258f85bf468b

**Comment by rirze** [comment]
Could you test this with the newest dev commit, namely any commit past `dc8b62`. There were issues with values not being loaded from the config properly.

**Comment by loveinpin** [comment]
Sure   Trying to upgrade dev now 

Reply result later

**Comment by destiny117** [comment]
i think there is some other issue, config values are loaded but it still uses pokeballs even at <15% chance

**Comment by HibikiFox** [comment]
Just lost a 97.5% venasaur because it didnt swap balls T_T

**Comment by joshpalan** [comment]
This was my commit. I believe I may know where the issue lies and am checking into it now

**Comment by joshpalan** [comment]
I believe it lies in lines 20-23 in inventory.py. Try changing it like so:

Before

        self.pokeball_percent = (percentages[0] / 100)
        self.greatball_percent = (percentages[1] / 100)
        self.ultraball_percent = (percentages[2] / 100)
        self.use_masterball = percentages[3]


After

        self.pokeball_percent = percentages[0] / 100
        self.greatball_percent = percentages[1] / 100
        self.ultraball_percent = percentages[2] / 100
        self.use_masterball = percentages[3]

**Comment by rirze** [comment]
Wait, are you doing integer division here?

**Comment by j-e-k** [comment]
Shouldn't the above be floating point division? The parenthesis don't matter, but it should be /100.0 instead of /100 (to avoid rounding...)

**Comment by j-e-k** [comment]
Does it work for you if you use floats instead of integers in the config? I.e `15.0` instead of `15`

**Comment by joshpalan** [comment]
@j-e-k I believe it was a rounding issue. I changed my local to divide by 100.0 instead and I believe it's working, though I've only encountered two pokemon so far. I'm keeping my eyes on it and will update my comment in the mean time

**Comment by joshpalan** [comment]
I noticed in #365 people were asking about the config. I'll update the readme as well in my next pr

**Comment by loveinpin** [comment]
yuh, rewrite point could solve this issue, bt pgoapi seems still not get info from config either, need 2 change Manually.

**Comment by destiny117** [comment]
@joshpalan fixed this with PR #388, open a new issue if you still run into problems
